indexing-service/src/main/java/org/apache/druid/indexing/common/actions/SegmentAllocationQueue.java

```diff
@@ -355,7 +353,7 @@ private boolean processBatch(AllocateRequestBatch requestBatch)
 
     emitBatchMetric("task/action/batch/attempts", 1L, requestKey);
     emitBatchMetric("task/action/batch/runTime", (System.currentTimeMillis() - startTimeMillis), requestKey);
-    log.info("Successfully processed [%d / %d] requests in batch [%s].", successCount, batchSize, requestKey);
+    log.debug("Successfully processed [%d / %d] requests in batch [%s].", successCount, batchSize, requestKey);
```

Comment:
is it too noisy?

Reply:
It is once per batch, so I guess we can go either way with this.